js/index.js

```diff
@@ -711,11 +420,13 @@ $(function() {
                     setupEditor();
                 }).fail(function() {
                     console.log("Unable to load gist contents");
+                    $('#code_textarea').val("-- Unable to load gist contents. Gist = " + id);
```

Comment:
I added it to make sure that it's not retrying in case of failure (and hitting github's ratelimit quickly as I found out)